doc/design/parallel-node-config.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Parallel SR-IOV configuration
+authors:
+  - e0ne
+reviewers:
+  - adrianchiris
+  - SchSeba
+creation-date: 18-07-2023
+last-updated: 18-07-2023
+---
+
+# Parallel SR-IOV configuration
+
+## Summary
+SR-IOV Network Operator configures NICs one by one. That means we’ll need to wait hours or even days to configure all
+NICs on large cluster deployments. With multi-NICs deployments Operator configures NICs one by one on each node which
+leads to a lot of unnecessary node drain calls during SR-IOV configuration.
+
+## Motivation
+Allow SR-IOV Network Operator to configure more than one node at the moment.
```

Comment:
I think thats the summary and what u got in summary is the motivation...

Reply:
nit: remove at the moment

Reply:
also add that we are moving all draining logic to a centralized place which will reduce chances of race conditions and bugs that were encountered before in sriov-network-config-daemon with draining.